package.json

```diff
@@ -19,7 +19,9 @@
     "lint": "eslint .",
     "start": "node src/shared/cli/index.js",
     "qa": "pta ./src/qa/*.qa.js -r log | node src/qa/utils/qaReporter.js",
-    "test": "FILES_MODIFIED=$(git diff --name-only HEAD) jest",
+    "test": "npm run test:jest && npm run test:tape",
+    "test:tape": "tape tests/**/*-test.js | tap-spec",
+    "test:jest": "FILES_MODIFIED=$(git diff --name-only HEAD) jest",
```

Comment:
Let's not have two test harnesses

Reply:
Of course. This is so folks could test things out.